docsets/Ramda/docset.json

```diff
@@ -1,13 +1,17 @@
 {
   "name": "Ramda",
-  "version": "0.24.1",
+  "version": "0.26.1",
   "archive": "Ramda.tgz",
```

Comment:
You need to update this archive as well, otherwise it will still point to the `0.24.1` docset.

Reply:
Ah, sorry about that. Updated.